### PR TITLE
SwitchOnOff Api

### LINKED ISSUE #38: SwitchOnOff API
Create an API to turn switch on or off.

### PR DESCRIPTION
**Pull request by misdaque**
Though completed yet it needs another walk through.
No roomId has been mentioned in the deviceStateLog table.
Should we use join table feature to do so?
Our Switch class contains a private variable 'off' of type boolean which is quiet confusing. If the value of off is 1 then is the device On or Off?
This off value I'm using to feed state column of deviceStateLog

### CODE REVIEW

File: src/main/java/com/autohome/model/Switch.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.autohome.model;
+
+
+public class Switch {
+
+    private boolean off;
```
Comment (keensam04) [inline src/main/java/com/autohome/model/Switch.java]: change name to on

File: src/main/java/com/autohome/controller/DeviceManagementController.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import com.autohome.model.Switch;
+import com.autohome.service.DeviceStateService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping(value = "/room/{roomId}/device/{deviceId}/operate")
+public class DeviceManagementController {
```
Comment (keensam04) [inline src/main/java/com/autohome/controller/DeviceManagementController.java]: Change name to DeviceOperationController

**Review by keensam04 — CHANGES_REQUESTED** [review]

File: src/main/java/com/autohome/model/Switch.java (hunk 1 above)
Comment (misdaque) [reply src/main/java/com/autohome/model/Switch.java]: Done...

File: src/main/java/com/autohome/controller/DeviceManagementController.java (hunk 2 above)
Comment (misdaque) [reply src/main/java/com/autohome/controller/DeviceManagementController.java]: Done

**Review by keensam04 — APPROVED** [review]

**Pull request merged**